api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1241,15 +1241,9 @@ public void setDescriptions(Collection<ConceptDescription> descriptions) {
 	 * @param description the description to add
 	 */
 	public void addDescription(ConceptDescription description) {
-		if (description != null && StringUtils.isNotBlank(description.getDescription())) {
-			if (getDescriptions() == null) {
-				descriptions = new HashSet<ConceptDescription>();
-				description.setConcept(this);
-				descriptions.add(description);
-			} else if (!descriptions.contains(description)) {
-				description.setConcept(this);
-				descriptions.add(description);
-			}
+		if (description != null && StringUtils.isNotBlank(description.getDescription()) && !descriptions.contains(description)) {
+		  	description.setConcept(this);
```

Comment:
@mavrk I still see a mix of whitespace and tabs. please replace whitespaces with tabs. configure your IDE to show you the whitespaces